src/pe_source/pe_scripts.py

```diff
@@ -0,0 +1,129 @@
+"""A tool for gathering pe source data.
+
+Usage:
+    pe-source DATA_SOURCE [--log-level=LEVEL] [--orgs=ORG_LIST] [--cybersix-methods=METHODS] [--soc_med_included]
+
+Arguments:
+  DATA_SOURCE                       Source to collect data from. Valid values are "cybersixgill",
+                                    "dnstwist", "hibp", "intelx", "dnsmonitor", and "shodan".
+
+Options:
+  -h --help                         Show this message.
+  -v --version                      Show version information.
+  -l --log-level=LEVEL              If specified, then the log level will be set to
+                                    the specified value.  Valid values are "debug", "info",
+                                    "warning", "error", and "critical". [default: info]
+  -o --orgs=ORG_LIST                A comma-separated list of orgs to collect data for.
+                                    If not specified, data will be collected for all
+                                    orgs in the pe database. Orgs in the list must match the
+                                    IDs in the cyhy-db. E.g. DHS,DHS_ICE,DOC
+                                    [default: all]
+  -csg --cybersix-methods=METHODS   A comma-separated list of cybersixgill methods to run.
+                                    If not specified, all will run. Valid values are "alerts",
+                                    "credentials", "mentions", "topCVEs". E.g. alerts,mentions.
+                                    [default: all]
+  -sc --soc_med_included            Include social media posts from cybersixgill in data collection.
```

Comment:
Underscores are not used for any of the other options:
```suggestion
  -sc --soc-med-included            Include social media posts from cybersixgill in data collection.
```

Reply:
Fixed in [d8e46a6](https://github.com/cisagov/pe-source/pull/1/commits/d8e46a6c95a0d185990945b7ba9d347bdeb1524c)